docs/advanced.rst

```diff
@@ -111,6 +111,25 @@ The following are valid arguments which may be passed to the Raven client:
 
         environment = 'staging'
 
+.. describe:: repos
+
+    This describes local repositories that are reflected in your source code::
+
+        repos = {
+            'raven': {
+                # the name of the repository as registered in Sentry
+                'name': 'getsentry/raven-python',
+                # the prefix where the local source code is found in the repo
+                'prefix': 'src',
+            }
+        }
+
+    The repository key can either be a module name or the absolute path. When
+    a module name is given it will be automatically converted to its absolute path.
+
+    For more information, see the :doc:`repos interface <../../../interfaces/repos>`
```

Comment:
aside, can we do like ``</interfaces/repos>`` rather than figuring out some magical relative path?